mRUBiS/ML_based_Control/src/de/mdelab/morisia/selfhealing/rules/ChunkedSocketCommunicator.java

```diff
@@ -0,0 +1,145 @@
+package de.mdelab.morisia.selfhealing.rules;
```

Comment:
can the empty files (mRUBiS/ML_based_Control/Logs/TrainingmRUBiS.csv) and mRUBiS/ML_based_Control/rulesToExecute.json) also be removed?

Reply:
Yes, you're right.